src/app/(cities)/bialystok/_FAQ.tsx

```diff
@@ -0,0 +1,35 @@
+import { FAQQuestion } from '@/components/FAQ';
+
+export const BialystokFAQ: FAQQuestion[] = [
+	{
+		id: '1',
+		question: 'How to become Białystok meetup sponsor or speaker?',
+		answer:
+			'Feel free to connect with one of the organizers on LinkedIn — we’d love to chat with you about sponsorship opportunities!',
```

Comment:
./src/app/(cities)/bialystok/_FAQ.tsx
15:7  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities